src/app/components/server-compare-charts/server-compare-charts.component.ts

```
@@ -0,0 +1,1076 @@
+    const dataSet = this.benchmarkMeta?.find((x: any) => x.benchmark_id === benchmark_id);
+
+    if(dataSet) {
+      let scales: any[] = [];
+      dataSet.configs.filter((x: any) => x.config[labelsField] === labelValue).forEach((item: any) => {
+        if((item.config[scaleField] || item.config[scaleField] === 0) && scales.indexOf(item.config[scaleField]) === -1) {
+          scales.push(item.config[scaleField]);
+        }
+      });
+
+      scales.sort((a, b) => {
+        if(!isNaN(a) && !isNaN(b)) {
```

Biome

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)